### ISSUE TITLE
Close Modal Button Text is Not Visible when Theme is chnaged to light

### ISSUE THREAD

**Issue by protikbiswas100** [microsoft/react-native-gallery#836]
“Close modal” button text is not visible inside the “This is a simple modal” dialog. 'Close modal' text color is mixed with the background color and unable to see it clearly. 'Close modal' button text is visible in the High contrast mode but normally it is not visible.

https://microsoft.visualstudio.com/OS/_workitems/edit/60966227/?view=edit
